src/content/en/fundamentals/instant-and-offline/web-storage/index.md

```diff
@@ -179,15 +192,16 @@ many browsers as possible and which offer asynchronous call models, to maximize
 interoperability with the UI. These criteria lead naturally to the following
 technology choices:
 
-* For device local key/value storage use the Cache API.
+* For offline storage, use the Cache API. This API is available in any browser that supports Service
```

Comment:
Small nit-- make Service Worker technology link to https://jakearchibald.github.io/isserviceworkerready/.